src/content/protocols/ethena/usde.md

```diff
@@ -0,0 +1,329 @@
+
+Multiple administrative permissions exist to modify this flow. The `DEFAULT_ADMIN_ROLE` in `EthenaMinting` can set or change the global mint and redeem caps using `setGlobalMaxMintPerBlock` and `setGlobalMaxRedeemPerBlock`. Furthermore, the `GATEKEEPER_ROLE` may call `disableMintRedeem`, which pauses all minting and redeeming operations. This function is designed to temporary preventing further minting or redeeming in the case of a vulnerability or instability. Users that have collateral deposited and are prohibited to access their collateral can lose funds if they have to sell it in the open market if the `USDe` trades below the collateral value. Restoring minting and redeeming functionality requires setting non-zero limits via the respective `setGlobalMaxMintPerBlock` and `setGlobalMaxRedeemPerBlock` functions which only can be executed by the `DEFAULT_ADMIN_ROLE`.
+
+The `GATEKEEPER_ROLE` may also revoke `MINTER_ROLE` and `REDEEMER_ROLE` from addresses that currently own the respective roles by calling `removeMinterRole`, `removeRedeemerRole` or `removeCollateralManagerRole`. Which is a designed security measure in case of compromised addresses.
+
+Additionally, `removeWhitelistedBenefactor` allows the `GATEKEEPER_ROLE` to remove an address from the whitelist. Only whitelisted addresses are allowed to supply valid orders. Ethena will not accept orders from addresses that are not whitelisted, to comply with AML laws and KYC requirements, as well as anti-terrorist financing laws. If an address created a position with collateral, but got removed from the whitelist, the address has to sell the `USDe` on the open market and has no access to its deposited collateral. This could lead to _loss of funds_ for users that have collateral deposited and are removed from the whitelist if the `USDe` price drops below the collateral value.
+
+The [Dev Multisig](#security-council) owns the `DEFAULT_ADMIN_ROLE` and can replace the `GATEKEEPER_ROLE` with another address and stays the ultimate on-chain authority over the `USDe` supply.
+
+Collateral flows to custodian addresses are executed through the `transferToCustody` function, callable only by addresses with the `COLLATERAL_MANAGER_ROLE`. The destination addresses for this function can only be added or removed by holding the `DEFAULT_ADMIN_ROLE`. Maliciously set addresses can lead to stealing collateral from users and the protocol.
+
+All core permissions for minting, redeeming, managing roles, and custodial flows are ultimately under the control of [Dev Multisig](#security-council), either directly or indirectly via its ability to control `DEFAULT_ADMIN_ROLE` holders.
```

Comment:
I find those paragraphs quite redundant, for instance everything mentioned here is said multiple times above